File: 이것이 취업을 위한 코딩 테스트다/구현/럭키_스트레이트/럭키_스트레이트.java
import java.util.Scanner;

public class 럭키_스트레이트 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String str = sc.next();

    int size = str.length();
    int mid = size / 2;
    int sum = 0;

    for (int i = 0; i < mid; i++) {
      sum += (str.charAt(i) - '0');
    }

    for (int i = mid; i < size; i++) {
      sum -= (str.charAt(i) - '0');
    }

    if (sum == 0) {
      System.out.println("LUCKY");
    } else {
      System.out.println("READY");
    }
  }
}

File: 이것이 취업을 위한 코딩 테스트다/구현/럭키_스트레이트/럭키_스트레이트.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
  string n;
  cin >> n;

  int size = n.size();
  int mid = size / 2;
  int sum = 0;

  for (int i = 0; i < mid; i++) {
    sum += (n[i] - '0');
  }

  for (int i = mid; i < size; i++) {
    sum -= (n[i] - '0');
  }

  if (sum == 0) {
    cout << "LUCKY";
  } else {
    cout << "READY";
  }

  return 0;
}